api/document/parsers.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+
+from lxml import etree
+from rest_framework.exceptions import ParseError
+from rest_framework.parsers import BaseParser
+
+from document.tree import PrimitiveDict
+
+
+def text(text: str) -> PrimitiveDict:

```

Comment:
Vanishingly small detail, but we use `text` to refer to strings in most of the code; should we make this `text_content` or something similar to distinguish?